andela-ifatoki/CP2-Shift-DMS
# #147515267 Setup server and endpoints

#### What does this PR do?
Setup a basic functional express server
#### Description of Task to be completed?
Setup a basic functional express server
#### How should this be manually tested?
- Start the server by running `npm run dev`
- You should get a prompt `database connected to 8000 successfully`
- Open your browser and navigate to `localhost:8000` you should get a response like this `{"message":"Welcome to the beginning of nothingness."}`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
147515267
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
